### PR TITLE
Add unsupported page for job conditions

### PR DESCRIPTION
**Pull request by ludeeus**
## Proposed change

Adds a more-info page for `job_conditions` unsupported reason.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase:  https://github.com/home-assistant/frontend/pull/7790
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/more-info/unsupported/job_conditions.markdown [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+## The issue
+
+Ignoring job conditions can be useful in a development setting, but running with this is not supported and it will cause issues, these issues can break your system.
+
+## The solution
+
+Remove all ignored conditions by using the CLI.
+
+```bash
+ha jobs options --ignore_conditions []
```
Comment (ludeeus) [inline source/more-info/unsupported/job_conditions.markdown]: This is not made yet, so the syntax could change

File: source/more-info/unsupported/job_conditions.markdown [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+Remove all ignored conditions by using the CLI.
+
+```bash
+ha jobs reset
+```
+
+When you have removed all of them, reload the Supervisor.
+
+```bash
+ha supervisor reload
```
Comment (pvizeli) [inline source/more-info/unsupported/job_conditions.markdown]:
```suggestion
ha supervisor restart
```

File: source/more-info/unsupported/job_conditions.markdown [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+## The solution
+
+Remove all ignored conditions by using the CLI.
+
+```bash
+ha jobs reset
+```
+
+When you have removed all of them, reload the Supervisor.
```
Comment (pvizeli) [inline source/more-info/unsupported/job_conditions.markdown]:
```suggestion
When you have removed all of them, restart the Supervisor.
```

**Review by frenck — APPROVED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

**Pull request merged**